runelite-client/src/main/java/net/runelite/client/ui/overlay/components/TextComponent.java

```diff
@@ -43,18 +49,50 @@
 	@Setter
 	private Color color = Color.WHITE;
 
+	@Setter
+	private boolean colTagsEnabled = true;
+
+	public static String textWithoutColTags(String text)
+	{
+		return COL_TAG_PATTERN.matcher(text).replaceAll("");
+	}
+
 	@Override
 	public Dimension render(Graphics2D graphics, Point parent)
 	{
-		// Draw shadow
-		graphics.setColor(Color.BLACK);
-		graphics.drawString(text, position.x + 1, position.y + 1);
+		final FontMetrics fontMetrics = graphics.getFontMetrics();
 
-		// Draw actual text
-		graphics.setColor(color);
-		graphics.drawString(text, position.x, position.y);
+		String[] parts = COL_TAG_PATTERN_W_LOOKAHEAD.split(text);
+		if (parts.length > 1 && colTagsEnabled)
```

Comment:
What are the use cases for having colTagsEnabled=false but having a <col>?

This also doesn't work right on messages with only one <col>